compiler/testData/codegen/bytecodeText/forLoop/forInReversed/forInReversedArray.kt

```diff
@@ -0,0 +1,63 @@
+// TARGET_BACKEND: JVM_IR
```

Comment:
NB you can actually do that directly in a box test, with 
```
// CHECK_BYTECODE_TEXT
// ... bytecode patterns as in bytecodeText tests ...
```

Reply:
Thanks, updated. And bytecode test `forInReversedArray.kt` has been removed  since it seems redundant now.